ci-operator/step-registry/redhat-chaos/hog-scenarios/memory/redhat-chaos-hog-scenarios-memory-commands.sh

```
@@ -46,5 +46,11 @@ rc=$?
 if [[ $TELEMETRY_EVENTS_BACKUP == "True" ]]; then
     cp /tmp/events.json ${ARTIFACT_DIR}/events.json
 fi
+
+
+if [[ -f /home/krkn/kraken/kraken.report.pdf ]]; then
+  cp /home/krkn/kraken/kraken.report.pdf ${ARTIFACT_DIR}/kraken.report.pdf
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)